src/lightning_lite/lite.py

```diff
@@ -160,6 +161,7 @@ def setup(
 
         # Let accelerator/plugin wrap and connect the models and optimizers
         model, optimizers = self._strategy.setup_module_and_optimizers(model, list(optimizers))
+        optimizers_to_device(optimizers, self.device)  # TODO: debugging, seems to make no difference
```

Comment:
debugging comment

Reply:
yeah, i wasn't finished debugging, but should be fixed now